lighthouse-core/audits/audit.js

```diff
@@ -78,10 +77,10 @@ class Audit {
   }
 
   /**
-   * @param {!Audit.Headings} headings
-   * @param {!Array<!Object<string, string>>} results
-   * @param {!DetailsRenderer.DetailsSummary} summary
-   * @return {!DetailsRenderer.DetailsJSON}
+   * @param {Audit.Headings} headings
+   * @param {Array<Object<string, string>>} results
+   * @param {Audit.DetailsRenderer.DetailsSummary} summary
```

Comment:
stuck these on `Audit` to keep them local since they're going to be pulled in by the larger LHR/lite effort. `summary` is defined as void for now to encourage adding types :)